apps/detect/src/components/dashboard/header.tsx

```diff
@@ -9,13 +12,56 @@ export default function Header({
   count: React.ReactNode;
   right?: React.ReactNode;
 }) {
+  const { data: reportData } = useQuery(queries.detect.activity.logs());
```

Comment:
We also need to apply the same filters to export

```suggestion
  const filters = useFiltersStore(selectQuery);
  const { data: reportData } = useQuery(queries.detect.activity.logs(filters));
```